src/hearings/containers/request-hearing/hearing-create-edit-summary/hearing-create-edit-summary.component.spec.ts

```diff
@@ -206,7 +206,7 @@ describe('HearingCreateEditSummaryComponent', () => {
     component.removeUnnecessarySummaryTemplateItems();
     fixture.detectChanges();
     const sfCreateEditStore = spyOn(component, 'getScreenFlowFromStore').and.returnValue(of(screenFlow));
-    component.getScreenFlowFromStore().subscribe((scr) => {
+    component.getScreenFlowFromStore().subscribe(() => {
```

Comment:
This test is quite poor - not your fault. Although you've removed an unused variable, all this does is make the test even less sensible. I guess I can live with it, but really the test should be redone to actually test the functionality, rather than just checking that static data is unchanged.